### DISCUSSION TITLE
Collate all "colour" issues under a single milestone

### DISCUSSION THREAD

**Discussion by savchenko** [taskchampion/taskchampion#215] [General]
Looking at the rapidly increasing size of [v0.5.0](https://github.com/taskchampion/taskchampion/milestone/2), wouldn't it make sense to move #86 and #116 under something like v0.5.1?

**Comment by djmitche** [comment]
That, or just tag them as "term-color" or something like that so that they're easy to move out to a later milestone if necessary.

**Reply by savchenko** [reply]
Done... Is there a way to "close" the discussion?

**Reply by dbr** [reply]
Nope not currently unless the question type is "Q&A",

https://github.com/github/feedback/discussions/3244
https://github.com/github/feedback/discussions/3097

Two workarounds suggested are:
1. Create an "Archive" category and move things there
2. Lock discussions and filter based on that (I think this requires typing in the filter each time?)

**Reply by savchenko** [reply]
3. Use "Issues" instead of "Discussions"?
